react/features/base/react/components/web/SectionList.js

```diff
@@ -63,23 +70,25 @@ export default class SectionList extends Component<Props> {
                 <Container
                     className = 'navigate-section-list'>
                     {
-                        sections.map((section, sectionIndex) => (
-                            <Container
-                                key = { sectionIndex }>
-                                { renderSectionHeader(section) }
-                                { section.data
-                                    .map((item, listIndex) => {
-                                        const listItem = {
-                                            item
-                                        };
+                        sections.length === 0
+                            ? ListEmptyComponent
+                            : sections.map((section, sectionIndex) => (
+                                <Container
+                                    key = { sectionIndex }>
+                                    { renderSectionHeader(section) }
+                                    { section.data
+                                        .map((item, listIndex) => {
+                                            const listItem = {
+                                                item
+                                            };
 
-                                        return renderItem(listItem,
-                                            keyExtractor(section,
-                                                listIndex));
-                                    }) }
-                            </Container>
-                        )
-                        )
+                                            return renderItem(listItem,
+                                                keyExtractor(section,
+                                                    listIndex));
+                                        }) }
+                                </Container>
+                            )
+                            )
```

Comment:
A loading icon somewhere would be nice. It can take 2+ seconds for the calendar data to load.